src/components/CodeView/styles.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../scss/variables';
+
+.CodeView {
+  border: 1px solid $light-gray;
+  padding: 10px;
+}
+
+.lineNumbers {
+  display: grid;
+  float: left;
+  padding-right: 10px;
+
+  * {
```

Comment:
github does `text-align: right` on line numbers which I think would be a good idea for when there are thousands of lines.

Reply:
Good idea, thanks!